src/main/java/com/fundation/webservice/controller/Controller.java

```diff
@@ -335,4 +335,146 @@ public KeyFrameResponse convertKeyFrame(@RequestParam("asset") MultipartFile ass
             return null;
         }
     }
+
+    public PPTtoPdfResponse PPTtoPdf(@RequestParam("asset") MultipartFile asset, @RequestParam("input") String input,
+                                     @RequestParam("config") String config, @RequestParam("output") String output) {
+        JSONObject inputJson = new JSONObject(input);
+        JSONObject configJson = new JSONObject(config);
+        JSONObject outputJson = new JSONObject(output);
+        String fileName = uploadService.storeFile(asset);
+        String fileDownloadUri = ServletUriComponentsBuilder.fromCurrentContextPath().path("/download/").path(outputJson.getString("name")
+                + ".zip").toUriString();
+        String inputChecksumString = "";
+        InputStream inputProperties;
+        try {
+            inputProperties = new FileInputStream("application.properties");
+            properties.load(inputProperties);
+        } catch (IOException io) {
+            io.printStackTrace();
+        }
+
```

Comment:
Remove this line.

Reply:
Adopted as this group standard, this line is inserted in order to make readibility easier. At this point we want to stress the difference between the operations carried before the line and after the line.
Conclusion: This blank line will be preserved.